### PR TITLE
ansible tower db

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: playbooks/ansible_tower.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+- hosts: ansible_tower
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../group_vars/ansible_tower/main.yml
+    - ../group_vars/researchdata/vault.yml
+  tasks:
+    - name: add postgres:10 appstream
```
Comment (carolyncole) [inline playbooks/ansible_tower.yml:9]: I wonder if you could just depend on the postgresql role, instead of having all these commands in here...

**Review by carolyncole — CHANGES_REQUESTED** [review]

File: playbooks/ansible_tower.yml (hunk 1 above)
Comment (kayiwa) [reply playbooks/ansible_tower.yml:9]: Ubuntu and Centos have varying ways of implementing postgres.  Will look into conditional logic.

File: playbooks/ansible_tower.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+- hosts: ansible_tower
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../group_vars/ansible_tower/main.yml
+    - ../group_vars/researchdata/vault.yml
```
[the remark this answers is not in the document]
Comment (kayiwa) [reply playbooks/ansible_tower.yml:7]: 🤦🏿

**Review by carolyncole — APPROVED** [review]
Nice!  I ❤️  leveraging roles!

**Pull request merged**